steward/src/cork.rs

```diff
@@ -94,6 +108,121 @@ impl steward::contract_call_server::ContractCall for CorkHandler {
     }
 }
 
+pub struct DirectCorkHandler;
+
+#[async_trait]
+impl steward::contract_call_server::ContractCall for DirectCorkHandler {
+    async fn submit(
+        &self,
+        request: Request<SubmitRequest>,
+    ) -> Result<Response<SubmitResponse>, Status> {
+        debug!("received contract call request");
+        let request = request.get_ref();
+
+        let contract_call_data = match request.call_data.clone() {
+            Some(call) => call,
+            None => {
+                return Err(tonic::Status::invalid_argument(
+                    "Error, can't find call data",
+                ))
+            }
+        };
+
+        let config = APP.config();
+
+        let name = &config.keys.delegate_key;
```

Comment:
We've been using `delegate_key` to refer to the Cosmos key in the configuration. It's unlikely that someone is intending to use the same key material for their Ethereum testing. Maybe the Etheruem key to use needs to be an option on the command line.